projects/novo-elements/src/elements/data-table/data-table.component.ts

```diff
@@ -17,7 +17,7 @@ import {
   ViewChild,
   ViewChildren,
 } from '@angular/core';
-import { Subscription } from 'rxjs';
+import {Subject, Subscription} from 'rxjs';
```

Comment:
can you fix the spacing with this import should be `import { Subject, Subscription } from 'rxjs';`

Reply:
updated